sample/src/test/kotlin/io/github/bluegroundltd/sample/MyEnumTest.kt

```diff
@@ -0,0 +1,30 @@
+package io.github.bluegroundltd.sample
+
+import org.junit.Test
+
+class MyEnumTest {
+
+    @Test
+    fun `should map string value to enum value`() {
+        // Given
+        val firstEntryValue = MyEnum.FIRST_ENTRY.value
+
+        // When
+        val firstEntryEnumValue = MyEnumMapper.fromValue(value = firstEntryValue)
+
+        // Then
+        assert(firstEntryEnumValue == MyEnum.FIRST_ENTRY)
+    }
+
+    @Test
+    fun `should map unknown string value to the enum value marked with @DefaultEnumConvertible`() {
```

Comment:
💡 Maybe we could add a test for the case when `@DefaultEnumConvertible` is not used